integration_test/tests/common/mod.rs

```diff
@@ -14,7 +14,7 @@ pub fn generate_default_connection_str() -> String {
     let db = env::var("ADF_TEST_LOCAL_DB");
     let driver = match env::var("ADF_TEST_LOCAL_DRIVER") {
         Ok(val) => val,
-        Err(_e) => "ADF_ODBC_DRIVER".to_string(), //Default driver name
+        Err(_e) => "MONGODB_ODBC_DRIVER".to_string(), //Default driver name
```

Comment:
Could we move this to constants? I'm working on the setupgui and will be using this a lot. I can do it as part of that PR if you want to leave as is for now.